### PR TITLE
fix(config): update Inovelli LZW60 device to better match upstream documentation

### PR DESCRIPTION
**Pull request by warthog9**
This is basically a merge of the LZW60 file with the one that
made it in already.  This better matches the documentation
provided by Inovelli here:

https://community.inovelli.com/t/device-page-inovelli-4-in-1-motion-sensor-lzw60/1840

Signed-off-by: John 'Warthog9' Hawley <warthog9@eaglescrag.net>

### CODE REVIEW

File: packages/config/config/devices/0x031e/lzw60.json [hunk 1]
```diff
@@ -30,19 +32,62 @@
...
+				{
+					"label": "9",
+					"value": 9
+				},
+				{
+					"label": "10 - High",
+					"value": 10
+				}
+			]
 		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw60.json]: I'd leave this on `allowManualEntry: true` and add a single option for disabled (0)

File: packages/config/config/devices/0x031e/lzw60.json [hunk 2]
```diff
@@ -30,19 +32,62 @@
 			"allowManualEntry": true
 		},
 		"12": {
-			"label": "PIR Sensitivity",
-			"description": "Change the sensitivity of the PIR (motion) sensor; 0 = off, 1 (low) - 10 (high)",
+			"label": "Motion Sensor Sensitivity",
+			"description": "Sensitivity level of the motion sensor. 0=Disabled 1=Low 10=High",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw60.json]:
```suggestion
			"description": "Sensitivity level of the motion sensor. 0 = Disabled, 1 = Low, 10 = High",
```

File: packages/config/config/devices/0x031e/lzw60.json [hunk 3]
```diff
@@ -110,7 +157,7 @@
 		},
 		"102": {
 			"label": "Humidity Reporting Interval",
-			"description": "Note: The set value will round to the nearest minute. (0 = Off)",
+			"description": "Interval, in seconds, in which humidity reports should be sent. 0=Disabled. Note: The set value will round to the nearest minute.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw60.json]: I don't like this change. We're trying to keep these short and not have redundancy with the label and the definitions (unit, etc.) below. Renaming "Off" to "Disabled" is okay IMO.

Comment (AlCalzone) [reply packages/config/config/devices/0x031e/lzw60.json]: Same goes for the others.

File: packages/config/config/devices/0x031e/lzw60.json [hunk 4]
```diff
@@ -136,19 +183,41 @@
...
+				},
+				{
+					"label": "Yes",
+					"value": 1
+				}
+			]
+		},
 		"111": {
 			"label": "Temperature Report Threshold",
+			"description": "Threshold for temperature reports to be sent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw60.json]: These are redundant IMO.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x031e/lzw60.json [hunk 5]
```diff
@@ -136,19 +183,41 @@
...
 			"maxValue": 2678400,
-			"defaultValue": 86400,
+			"defaultValue": 7200,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
+		"110": {
+			"label": "Send Reports According to Threshold",
```
Comment (blhoward2) [inline packages/config/config/devices/0x031e/lzw60.json]: We're also trying to remove yes/no options. Please make this something like "Report Only When Threshold's Are Met". And make the options Enable/Disable.

**Review by blhoward2 — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
@warthog9 do you plan to continue working on this?

**Comment by warthog9** [comment]
> @warthog9 do you plan to continue working on this?

Yes, sorry, just hasn't percolated back up on my todo list yet. Probably have some progress before the end of the week the way it's looking right now though

**Comment by warthog9** [comment]
Ok updated stuff, adjusted some of the wording as requested, looks ok (wording wise and size on screen) to me on my system right now

**Comment by blhoward2** [comment]
@warthog9 Earlier today we merged detailed documentation on device files, including a style guide and detailed guidance on templating. Could you please take a look at those, make any changes necessary to conform, and then let us know when you're ready for us to take a look line-by-line?

https://zwave-js.github.io/node-zwave-js/#/

**Comment by warthog9** [comment]
> @warthog9 Earlier today we merged detailed documentation on device files, including a style guide and detailed guidance on templating. Could you please take a look at those, make any changes necessary to conform, and then let us know when you're ready for us to take a look line-by-line?
> 
> https://zwave-js.github.io/node-zwave-js/#/

Ok made a couple more edits to try and follow the style guide. One point of clarification, I originally had an explicit option list with the 10 values the PIR sensitivity would be valid for (admittedly I had forgotten the disabled 0 state), as requested I switched it back to a manual entry.  By my reading of the style guide it seems like it should go back to an option list, or is my reading of that incorrect?

**Comment by blhoward2** [comment]
10 options is starting to be a lot. If it is just 0-10 as a range I'd leave it manual. If the 0-10 mean other things (0 = blue, 1 = red, etc.) then I'd define them. Otherwise its just a drop-down with 10 choices--1, 2, 3, etc.

**Comment by blhoward2** [comment]
To be clear, though, you need to look at the templating document too. We don't yet have a manufacturer-specific one for this manufacturer yet so you'll need to create one, move parameters that also exist on other devices to the template from this file, use the master-level bases, etc.

File: packages/config/config/devices/0x031e/lzw60.json [hunk 6]
```diff
@@ -30,19 +32,26 @@
...
 			"maxValue": 10,
 			"defaultValue": 8,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "0 - Disable",
```
Comment (blhoward2) [inline packages/config/config/devices/0x031e/lzw60.json]: Take another look at the style guide. This should just say Disable.

File: packages/config/config/devices/0x031e/lzw60.json [hunk 7]
```diff
@@ -65,17 +75,18 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Disabled",
+					"label": "No",
```
Comment (blhoward2) [inline packages/config/config/devices/0x031e/lzw60.json]: There is a specific comment about avoiding yes/no, converting to enable/disable, and using the approbation base.

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Just some quick notes to get you started.

File: packages/config/config/devices/0x031e/lzw60.json (hunk 7 above)
Comment (warthog9) [reply packages/config/config/devices/0x031e/lzw60.json]: Sorry, I apparently amended the commit and pushed, but had forgotten to add the file I was committing with, should be fixed

**Comment by AlCalzone** [comment]
FYI, no need to amend the commits. We're squash-merging anyways.

**Review by blhoward2 — APPROVED** [review]
I made some additional minor changes. This one isn't templated but given how long its been open, lets just merge it and I'll template it when I get to this folder.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**